eng/pipelines/templates/jobs/ci.yml

```diff
@@ -191,6 +190,44 @@ jobs:
           failOnStderr: false
       - template: /eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml
 
+  - job: Compliance
+    pool:
+      name: azsdk-pool-mms-win-2019-general
+      vmImage: MMS2019
+    steps:      
+      - pwsh: |
+          New-Item -Path "$(Build.sourcesdirectory)" -Name "credscan.tsv"
+          if ("$(Build.Reason)" -eq 'PullRequest') {
+            New-Item -Path "$(Build.sourcesdirectory)" -Name "changedFiles" -ItemType "directory"
+            Write-Host "origin/$(System.PullRequest.TargetBranch)"
+            (git diff "origin/$(System.PullRequest.TargetBranch)" HEAD --name-only) | ForEach-Object { Copy-Item $_ -Destination "$(Build.sourcesdirectory)\changedFiles"}
```

Comment:
I would still like to see wat this looks like if we don't copy the files but instead just scan all the directories?

Reply:
1. PR validation (**5 mins** run): https://dev.azure.com/azure-sdk/public/_build/results?buildId=1327591&view=logs&jobId=bc67675d-56bf-581f-e0a2-208848ba68ca
2. Ci build(Storage) (**15 mins**): [to be continued](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1327596&view=logs&j=bc67675d-56bf-581f-e0a2-208848ba68ca)

Reply:
@weshaggard 

Storage takes 4 mins longer than build job. What do you think about it?